### PR TITLE
Enable CFT Lint bot

### PR DESCRIPTION
**Pull request by bharathkkb**
- Adds CFT lint workflow to this module

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -69,6 +69,15 @@ docker_test_lint:
 		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
 		/usr/local/bin/test_lint.sh
 
+# Execute lint tests in github actions
+.PHONY: docker_test_lint_gha
+docker_test_lint_gha:
+	docker run --rm \
+		-v $(CURDIR):/workspace \
+		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
+		/usr/local/bin/test_lint.sh --markdown --contrib-guide=https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/master/CONTRIBUTING.md
```
Comment (morgante) [inline Makefile:78]: Can we make this a relative path?

Comment (bharathkkb) [reply Makefile]: So I think relative paths for PRs are still buggy: https://github.com/github/markup/issues/576
I was able to do a workaround. PTAL

Comment (bharathkkb) [reply Makefile]: Example with relative path staged here: https://github.com/bharathkkb/terraform-google-cloud-storage/pull/8

Comment (morgante) [reply Makefile]: Great, let's also make that the default (in the central script) so we don't have to specify it for each repo.

**Pull request merged**